scripts/lib/k8s.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+ensure_controller_gen() {
+  go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
+}
+
+ensure_service_controller_running() {
+  echo "Ensure that service controllers are running for given version"
+  # Give executable permission
+  chmod +x ./scripts/generate-crds.sh
+
+  __image_version=$1
+  __crd_path=$2
+
+  for d in ./services/*; do
+    if [ -d "$d" ]; then
+      echo "service: $d"
```

Comment:
best to remove the above echo. I presume it was just there for debugging purposes while you wrote this script?